### DISCUSSION TITLE
Using a transitional element with the GitHub Inverto Cat logo

### DISCUSSION THREAD

**Discussion by seantant** [community/community#135266] [Other Feature Feedback, Questions, & Ideas]
I want to use the Inverto cat logo on my website to link to my GitHub account, in addition to a circular background that is flush with the logo's circular edge, and transitions from transparent to a color that is different from the main background, so the negative space turns a different color. In effect, the cat itself should change color, but the color of the logo PNG remains untouched. My question is, would this be considered changing the color of the logo or would this be permitted use? I feel like this is a bit of a gray area and am unsure of how to interpret the legal terms.

**Comment by dane** [comment]
As a user of GitHub, not an employee, what you're describing sounds like it'd be fine because you're only changing colors. NPM (npmjs.com, acquired by GitHub) uses a black and white GitHub logo. Rust (rust-lang.com) uses a white and black GitHub logo. It's pretty common to link back to GitHub and have the logo fit the color scheme of your website.

I wouldn't recommend using a PNG or any other image format. If you inspect the GitHub logo in the footer, you'll see it's an SVG. Using an SVG will shrink and scale infinitely and you can control the colors via CSS.

Hope this helps!

**Reply by seantant** [reply]
Hi dane, I appreciate your response. The reason I ask is because some of the guidelines outlined [here](https://github.com/logos) say that changing the color (outside of the black or white they give you) is not allowed. Particularly, I'm concerned that I could be violating this bullet:

> "Do not modify the permitted GitHub logos, including changing the color, dimensions, or combining with other words or design elements."

Even though I am not modifying the logo itself, I am wondering if coloring the negative space would be considered changing the color or adding design elements.

Also PNG is what they provide, but I will try SVG. I'm sure as long as I follow the same rules it should be fine to use that instead of a PNG. It was quite low quality so that should be pretty helpful.


Thanks so much!
